bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "adapt-contrib-resources",
-  "version": "4.0.0",
+  "version": "4.0.1",
   "framework": ">=5",
```

Comment:
FW needs to be at least 5.3 if you want to use ES6-8 code; I've generally been setting these to >=5.5 since that's when we started converting the code over.